build/AspNetCore.slnf

```diff
@@ -0,0 +1,9 @@
+{
+  "solution": {
+    "path": "..\\DotNetWorker.sln",
+    "projects": [
+      "extensions\\Worker.Extensions.Http.AspNetCore\\src\\Worker.Extensions.Http.AspNetCore.csproj",
+      "extensions\\Worker.Extensions.Http.AspNetCore.Analyzers\\Worker.Extensions.Http.AspNetCore.Analyzers.csproj"
+    ]
```

Comment:
Creating a solution filter for AspNetCore similar to [DotNetWorker.Core.slnf](https://github.com/Azure/azure-functions-dotnet-worker/blob/main/build/DotNetWorker.Core.slnf) so that the pipeline creates packages for these projects.